assets/js/googlesitekit/widgets/util/combine-widgets.test.js

```diff
@@ -30,13 +30,64 @@ import Null from '../../../components/Null';
 describe( 'combineWidgets', () => {
 	const getQuarterWidget = ( slug ) => ( { slug, width: WIDGET_WIDTHS.QUARTER } );
 	const getHalfWidget = ( slug ) => ( { slug, width: WIDGET_WIDTHS.HALF } );
+	const getFullWidget = ( slug ) => ( { slug, width: WIDGET_WIDTHS.FULL } );
 
 	const getRegularState = () => null;
 	const getReportZeroState = ( moduleSlug ) => ( { Component: ReportZero, metadata: { moduleSlug } } );
 	const getActivateModuleCTAState = ( moduleSlug ) => ( { Component: ActivateModuleCTA, metadata: { moduleSlug } } );
 	const getCompleteModuleActivationCTAState = ( moduleSlug ) => ( { Component: CompleteModuleActivationCTA, metadata: { moduleSlug } } );
 	const getNullState = () => ( { Component: Null, metadata: {} } );
 
+	it( 'Widgets in an area are combined beyond their row if all of them have the same special state', () => {
```

Comment:
Test names should read as a sentence starting with "it ..." but these do not. A minor detail but consistency is important.